Buildings/Fluid/AirFilters/BaseClasses/Data/Generic.mo

```diff
@@ -0,0 +1,49 @@
+within Buildings.Fluid.AirFilters.BaseClasses.Data;
+record Generic "Generic data record for air filters"
+  extends Modelica.Icons.Record;
+  parameter Real mCon_nominal(
+    final unit = "kg")
+    "Maximum mass of the contaminant can be captured by the filter";
+  parameter String substanceName[:] = {"CO2"}
+    "Name of trace substance";
+  parameter
+    Buildings.Fluid.AirFilters.BaseClasses.Characteristics.filtrationEfficiencyParameters
+    filterationEfficiencyParameters
+    "Filtration efficiency vs. relative mass of the contaminant";
+  parameter Real b = 1.1
+    "Resistance coefficient";
+  annotation (
+  defaultComponentPrefixes = "parameter",
+  defaultComponentName = "per",
+  Documentation(revisions="<html>
+<ul>
+<li>
+June 27, 2023, by Sen Huang:<br/>
+First implementation.
+</li>
+</ul>
+</html>", info="<html>
+<p>
+Record containing performance parameters for air filters.
+</p>
+<p>
+It is used as a template of performance data
+for the variable-speed wheel models in
+<a href=\"modelica://Buildings.Fluid.AirFilters\">
+Buildings.Fluid.AirFilters</a>.
+</p>
+<p>
+The record contains one dataset: 
+relative mass of the contaminant
+(see <a href=\"modelica://Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency\">
```

Comment:
is it the right link to the right class? I think it should like to `Buildings.Fluid.AirFilters.BaseClasses.Characteristics.filtrationEfficiencyParameters`?

Reply:
I meant the definition of the relative mass is provided in <code>modelica://Buildings.Fluid.AirFilters.BaseClasses.FiltrationEfficiency</code>.
I update the document to avoid confusions.